### DISCUSSION TITLE
Has ConfigBuilder been removed?

### DISCUSSION THREAD

**Discussion by pjjohnston** [open-telemetry/opentelemetry-java#3043] [Q&A]
Hi,
I'm having an issue building in Maven - 

The following code will not compile:

SimpleSpanProcessor processor = SimpleSpanProcessor.builder(exporter).build();

The builder above extends  ConfigBuilder which appears to have been removed from io.opentelemetry.sdk.common.export

Any help would be greatly appreciated!

Using the following Maven dependencies:

opentelemetry-bom 1.0.1
opentelemetry-api 0.13.1
opentelemetry-sdk-tracing 0.11.0
opentelemetry-exporters-logging 0.9.1
opentelemetry-api-common 0.13.1

**Comment by jkwatson** [comment]
You can't mix & match versions like that. You need to use the bom to keep all the versions in sync with each other.

**Reply by anuraaga** [reply]
Yeah @pjjohnston - you shouldn't need to specify a version number for any of those artifacts except for the `-bom` one.

**Comment by jkwatson** [comment]
Also, yes, the ConfigBuilder has certainly been removed, to answer the original question.

**Comment by pjjohnston** [comment]
Thanks guys - searching on Maven central was led me to believe that versions above were the latest for those artifacts. I see from the BOM that all versions are up to 1.0.1
So this got rid of the issue with the transitive dependency

**Reply by jkwatson** [reply]
Maven central search is really awful. Sorry this caused confusion.

**Reply by pjjohnston** [reply]
Don't mention it! Thanks for replying

**Comment by hobofan** [comment]
For anyone else stumbling upon this issue: It seems that at some point the exporter packages were renamed from `opentelemetry-exporters` (note the trailing `s`) to `opentelemetry-exporter`.

E.g. [opentelemetry-exporters-jaeger](https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-exporters-jaeger) -> [opentelemetry-exporter-jaeger](https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-exporter-jaeger), where you will encounter this issue if you try to use the older one (which sadly still shows up in top hits on Google when searching the contained classes).
